src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -200,10 +211,16 @@ protected void buildEndpoint(ApiEndpoint baseEndpoint) {
             // OtpUserController. Therefore, the request should be limited to return just the entity matching the
             // requesting user.
             return persistence.getResponseList(Filters.eq("_id", requestingUser.otpUser.id), offset, limit);
```

Comment:
If a third party API user if requesting a list of OTP users, the conditional block will end up here, right? And that will end up with a NPE on `requestingUser.otpUser.id` if I'm correct.

Reply:
Updated and included a matching unit test.